src/sdk/pynni/nni/utils.py

```diff
@@ -216,3 +217,16 @@ def json2parameter(x, is_rand, random_state, oldy=None, Rand=False, name=NodeTyp
     else:
         y = copy.deepcopy(x)
     return y
+
+class ClassArgsValidator(object):
```

Comment:
this class is supposed to be used by users, better to add docstring for it

Reply:
Thanks, docstring added.